doc/source/Linux.rst

```diff
@@ -0,0 +1,190 @@
+Linux Tutorial
+==============
+
+This guide gives you a brief introduction to how volatility3 works and some demonstration on suite of plugins available from
+
+Acquiring memory
+----------------
+
+Volatility does not provide the ability to acquire memory. In this tutorial we will see how we can use  `LiME <https://github.com/504ensicslabs/lime>`_ for this purpose. 
+It supports 32 and 64 bit captures from native Intel hardware as well as virtual machine guests. 
+It also supports capture from Android devices. See below for example commands building and running LiME:
+
+.. code-block:: shell-session
+
+    $ tar -xvzf lime-forensics-1.1-r14.tar.gz 
+    $ cd lime-forensics-1.1-r14/src
+    $ make
+    ....
+      CC [M]  /home/mhl/Downloads/src/tcp.o
```

Comment:
Might be better to use relative paths, since this doesn't seem to be under the just extracted directory (`lime-forensics-1.1-r14` isn't mentioned anywhere)